src/finch/tensor.py

```diff
@@ -304,6 +310,12 @@ def _from_numpy(cls, arr: np.ndarray, fill_value: np.number) -> JuliaObj:
             lvl = Dense(lvl, arr.shape[i])
         return jl.swizzle(jl.Tensor(lvl._obj), *order)
 
+    @classmethod
+    def from_scipy_sparse(cls, x) -> "Tensor":
+        if not _is_scipy_sparse_obj(x):
+            raise ValueError("{x} is not a SciPy sparse object.")
+        return Tensor(x)
```

Comment:
It looks like Tensor does not attempt to convert scipy formats. Could we similarly copy the format if a zero-copy constructor is not possible here?

Reply:
Currently `finch-tensor` accepts three SciPy formats in a constructor: `COO`, `CSR`, and `CSC`. All three formats are consumed in a zero-copy way.

What do you mean by `similarly copy the format if a zero-copy constructor is not possible`?
To convert SciPy's other formats (DOK, BSR) to COO first with SciPy API?

(I call `Tensor(x)` here because it already knows how to process scipy input. `def from_scipy_sparse` is only for API compliance purposes)

Reply:
Yes, let's convert SciPy's other formats (DOK, BSR) to COO first with SciPy API.

Reply:
eventually we can convert BSR but we'll need some compiler improvements

Reply:
Done!